package.json

```diff
@@ -1,14 +1,22 @@
 {
   "name": "normandy-nextgen-study-example",
-  "version": "0.3",
+  "version": "0.3.0",
   "description": "Demonstrates how to make a 2019-style Normandy add-on study",
   "private": true,
   "license": "MPLv2",
   "scripts": {
     "build": "node build.js",
-    "clean": "rimraf dist web-ext-artifacts"
+    "clean": "rimraf dist web-ext-artifacts",
+    "eslint": "eslint . --ext jsm,js,json",
+    "format": "npm run prettier && npm run eslint -- --fix",
```

Comment:
I think we ought to make this `yarn prettier && yarn eslint --fix` instead, since yarn is used elsewhere in the project. Alternatively, we could use something like [npm-run-all](https://www.npmjs.com/package/npm-run-all) which has some more features and will intelligently pick between npm and yarn.